### DISCUSSION TITLE
Idea for new season + new mechanics

### DISCUSSION THREAD

**Discussion by Dionis404** [sunflower-land/sunflower-land#3728] [Ideas]
### Culinary Madness

**New Season Idea**
This new idea for one of the game's seasons will radically change the mechanics of cooking and leveling up Bumpkin.

**New Cooking Mechanic**
Instead of providing players with ready-made recipes, this proposal allows them to combine various ingredients to create unique dishes. For example, to make pancakes, a player will need to mix three ingredients to get their dish.

This mechanic is somewhat reminiscent of flower breeding but involves more complex combinations and a wider variety of options.

**Types of Food**
Raw food: salads, slices, sushi, smoothies, juices.
Cooked food: fried and boiled dishes.
Baked goods: bread, cakes.
Preserved food: sauerkraut, pickled carrots.
Magical dishes?
_Fish cannot be eaten immediately after being caught; it will need to be processed._

**Single Cooking Building**
To simplify the interface, a single cooking building can be created with the ability to level up and unlock new features.

**Using Bumpkins to Unlock Cooking Slots?**
Each Bumpkin can unlock one cooking slot.

**Using Resources for Cooking**
For recipes requiring cooking over fire, wood will be needed to create the fire. Later, oil can be used to keep the fire burning longer.

**Quality of Prepared Food**
Food can be of different quality levels:

- Common
- Good
- Delicious
- Exquisite
- Divine
Quality affects the price of the dish for delivery and the XP it gives.

**Player-Friendly Mechanic**
Although the mechanic may seem complex, all recipes will be available in the codex after they are discovered. Recipes can be unlocked through experimentation or by finding them in the plaza. Once a recipe is unlocked, it can be selected, and the ingredients will be automatically chosen.

**Season Storyline**
A Bumpkin from distant lands brings a unique recipe and spices that have enthralled all the residents of Sunflower Land. Now, they want to try various dishes from around the world.

**Seasonal Quest**
The seasonal quest will involve preparing several dishes for a weekly feast. Players need to prepare and deliver 4-6 dishes to NPCs in exchange for seasonal tickets.

**Spices**
Spices are a new type of crop. They are not mandatory ingredients for preparing dishes but increase the chance of producing higher-quality food.

**Comment by LTNGx39** [comment]
I like the idea, unfortunately it is very complex to reform the entire food mechanic at the current point of the game, even so it may be implemented in the future.
**It would give a new touch to the cooking**

**Reply by Dionis404** [reply]
Yes, I totally agree that adding such mechanics would be difficult. 
That's why I propose to make it part of one of the new seasons, so that the team can throw all their energies into improving cooking in the game.
